tests/nlp/nlp_conftest.py

```diff
@@ -36,3 +39,31 @@ def text_multilabel_classification_dataset_mock():
     return TextData(['I think therefore I am', 'I am therefore I think', 'I am'],
                     [[0, 0, 1], [1, 1, 0], [0, 1, 0]],
                     task_type='text_classification')
+
+
+@pytest.fixture(scope='session')
+def movie_reviews_data():
+    """Dataset of single sentence samples."""
+    sentences = [' '.join(x) for x in movie_reviews.sents()]
+    random.seed(42)
+    train_data = TextData(random.choices(sentences, k=10000))
```

Comment:
Why use 10K here and not 1K like the two other fixtures?

Reply:
@noamzbr because there's not enough overlap in the random split to show that it's more similar than split by label with less then 10k